app/views/boards/show.html.erb

```diff
@@ -56,11 +48,25 @@ See doc/COPYRIGHT.rdoc for more details.
   <% end %>
 </div>
 
-<h2><%=h @board.name %></h2>
-<p class="subtitle"><%=h @board.description %></p>
-<%= render partial: 'layouts/action_menu_specific' %>
-
-
+<%= toolbar title: @board.name, subtitle: @board.description do %>
+  <% if authorize_for(:messages, :new) %>
+    <li class="toolbar-item">
+      <%= link_to({ controller: '/messages', action: 'new', board_id: @board},
+                    class: 'button -alt-highlight',
+                    onclick: 'Element.show("add-message"); Form.Element.focus("message_subject"); return false;') do %>
+        <i class="button--icon icon-add"></i> <%= l(:label_message_new) %>
+      <% end %>
+    </li>
+  <% end %>
+  <% unless User.current.nil? %>
```

Comment:
For the anonymous user I am getting an empty button...

![empty-button](https://cloud.githubusercontent.com/assets/453584/7720515/7ecd73a0-fece-11e4-8bc2-754b7db44747.PNG)